mindsdb/integrations/handlers/auto_ts_handler/README.md

```diff
@@ -0,0 +1,99 @@
+### Briefly describe what ML framework does this handler integrate to MindsDB, and how?
+Auto_timeseries is a complex model building utility for time series data. Since it automates many 
+Tasks involved in a complex endeavor, it assumes many intelligent defaults. But you can change them. 
+Auto_Timeseries will rapidly build predictive models based on Statsmodels ARIMA, Seasonal ARIMA, Prophet 
+and Scikit-Learn ML. It will automatically select the best model which gives best score specified.
+
+
+
+Call this handler with
+`USING ENGINE='auto_ts'` 
+
+### Why is this integration useful? What does the ideal predictive use case for this integration look like? When would you definitely not use this integration?
+This integration is useful for time series data. It will automatically select the best model which gives best score specified. And it integrations with best Time Series 
+libraries like Statsmodels ARIMA, Seasonal ARIMA, Prophet and Scikit-Learn.
+
+If there is more than one target variable in your data set, just specify only one for now, and if you know the time interval that is in your data, you can specify it.
+Otherwise it auto-ts will try to infer the time interval on its own
+
+### Are models created with this integration fast and scalable, in general?
+The model training time is fast and scalable as it uses best Time Series libraries like Statsmodels ARIMA, Seasonal ARIMA, Prophet and Scikit-Learn ML.
+As Auto_ts uses doesn't use neural networks, it is not prone to overfitting and take less time to train.
+
+
+### What are the recommended system specifications for models created with this framework?
+No CUDA enabled GPU is required. It can be run on any machine. 
+Some users can face issues while installing the package, [refer](https://github.com/AutoViML/Auto_TS#install)
+
+### To what degree can users control the underlying framework by passing parameters via the USING syntax?
+Almost all the inputs that Auto_TS takes can be passed via the USING syntax. 
+
+
+``` sql
+USING
+    engine='auto_ts',
+    target = 'Sales',
+    ts_column = 'Time_Period',
```

Comment:
Both `ts_column` and `target` columns should not be passed via `USING`, but through `ORDER BY` and `PREDICT` respectively. Please check this is the case, and also modify the README.

Reply:
To be clear: examples below show usage of `PREDICT`, but the user should not need to repeat this name in `USING` later on.